app/css/main.css

```diff
@@ -232,3 +232,22 @@ fieldset.inline2 {
 }
 
 
```

Comment:
The whole CSS should be moved into css/jsonforms.css. The main.css only acts as the CSS for our demo, hence this would not be applied anywhere but the demo.